src/arizona_html.erl

```diff
@@ -31,7 +30,3 @@ to_safe(V) when is_integer(V) ->
     integer_to_binary(V, 10);
 to_safe(V) when is_float(V) ->
     io_lib:format("~p", [V]).
-
--spec safe_types() -> [binary | atom | integer | float].
-safe_types() ->
-    [binary, atom, integer, float].
```

Comment:
Where did this go?

Reply:
It is no longer necessary as it was used in the old compiler.